README.md

```diff
@@ -1,2 +1,24 @@
 # rest-api-express
-Management web app
+
+WEB APPLICATION FOR PRODUCT MANAGEMENT  
+  
+**Technologies:**  
++ NodeJS Express (including dependencies)  
++ MySQL  
++ JavaScript  
++ Html  
++ CSS  
+
+**Run server in terminal:**  
+```
+npm run dev
+```
+
+**You will need an user and a db in MySQL (XAMMP), and enter the data in a .env file with the variables: (the vars will link with the file: config/mysql_db)**  
++ DB_HOST=  
```

Comment:
People usually creates a `.env.example` file with these properties, so any one who needed can just copy the file, rename it and fill it up with the real information